src/main/proto/spine/options.proto

```diff
@@ -1003,3 +1006,23 @@ message CompareByOption {
     // the lower, enums — from the last number value to the 0th value, etc.
     bool descending = 2;
 }
+
+// Defines the error message used if a `set_once` field is set again.
+//
+// Applies only to the fields marked as `set_once`.
+//
+// Example: Using the `(set_once)` option.
+//
+//     message User {
+//         UserId id = 1 [(set_once) = true,
+//                        (if_set_again).error_msg = "The student ID can't be re-assigned."];
+//     }
+//
+message IfSetAgainOption {
+
+    // The default error message for the field.
```

Comment:
Which field? How about telling something like this

```proto
// The default error message used when the field with the `(set_once)` option is re-assigned.
```

I'm not insisting on this text, but we need to help the user more.